sormas-backend/src/main/java/de/symeda/sormas/backend/event/eventimport/EventImportFacadeEjb.java

```diff
@@ -278,12 +280,16 @@
 						importError = exception.getMessage();
 						break;
 					} else if (exception instanceof InvalidColumnException) {
-						throw (InvalidColumnException) exception;
```

Comment:
Doesn't this break the existing logic? I believe there should be an exception thrown here?

Reply:
throwing InvalidColumnException is removed to allow importing exported events. It's done similarly for cases/contacts and event participants